src/screens/SUE/SueReportScreen.tsx

```diff
@@ -155,7 +138,56 @@ export const SueReportScreen = ({ navigation }: { navigation: any }) => {
       .finally(() => setIsLoading(false));
   };
 
+  /* eslint-disable complexity */
+  const alertTextGeneratorForMissingData = () => {
+    switch (currentPage) {
```

Comment:
`currentProgress` please as used in the component instead of a different name here for the same value.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/978/commits/49b11db13461321ba143f833d4bb3ad49e52ca8d